x/fundraising/keeper/genesis.go

```diff
@@ -15,6 +15,14 @@ func (k Keeper) InitGenesis(ctx sdk.Context, genState types.GenesisState) {
 		panic(err)
 	}
 
+	// Prevents from nil slice
+	if genState.Params.AuctionCreationFee == nil || len(genState.Params.AuctionCreationFee) == 0 {
```

Comment:
This could be shortened to:

```suggestion
	if len(genState.Params.AuctionCreationFee) == 0 {
```

Reply:
I add the nil check after referencing [the code](https://github.com/cosmosquad-labs/squad/blob/main/x/liquidstaking/keeper/genesis.go#L16) in the `liquidstaking` module. Do you think the suggested change will be okay?

Reply:
I'm pretty sure that `len(slice) == 0` is `true` for nil slices, so the latter condition already includes the former one.